contentcuration/contentcuration/utils/asynctask.py

```diff
@@ -0,0 +1,68 @@
+import ast
+import logging as logger
+
+from celery import states
+from celery import Task as celeryTask
```

Comment:
Small nitpick, can we match the Python use of Pascal case for class names, both when defining new classes and renaming imports?

Reply:
yes! thank you for the suggestion! I will push a commit for that